task/opm-render-bundles/0.1/opm-render-bundles.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: opm-render-bundles
+spec:
+  description: Create a catalog index and render the provided bundles into it
+  params:
+    - name: binary-image
+      description: Image in which to build catalog
+      default: registry.redhat.io/openshift4/ose-operator-registry:latest
+    - name: bundle-image
+      description: Comma separated list of bundles to add
+    - name: operator-name
+      description: Name of the Operator
+    - name: operator-version
+      description: Version of the Operator
+    - name: default-channel
+      description: The channel that subscriptions will default to if unspecified
+      default: stable
+  workspaces:
+    - name: source
+      description: Workspace with the source code
+  steps:
+    - name: opm-render-bundles
+      image: "registry.redhat.io/openshift4/ose-operator-registry:latest"
```

Comment:
Maybe we can use the `konflux-test` image and have a function that fetches the appropriate `opm` version from a mapping like https://gitlab.cee.redhat.com/ansible-playbooks/iib-playbooks/-/blob/master/inventory/group_vars/iib_workers_prod?ref_type=heads#L12? 

This will probably also require a `ocp_version` parameter so we can pick the appropriate opm version?

Reply:
@arewm thoughts?

Reply:
I know that you have wanted to have a common mapping available for OPM versions. Would you have a proposed location for this mapping?

This would be a potential resolution to the comment below: https://github.com/konflux-ci/build-definitions/pull/1170#discussion_r1693169567

Reply:
AFAIU this isn't a problem for us as the version difference relates to console UI metadata so maybe we can go ahead and merge this for now, following up with an improvement once we understand how this mapping should work.